app/models/drupal_node_revision.rb

```diff
@@ -114,4 +114,9 @@ def render_body_email
     body
   end
 
+  def body_preview
+	  newBody = self.body.gsub(/^#+.+/, "")
```

Comment:
Sorry to nitpick -- would you mind using two-spaces as your indent, rather than tabs? It's just a little thing but it does help us keep our code tidy.